pom.xml

```diff
@@ -57,11 +57,15 @@
     <ver.libthrift>0.10.0</ver.libthrift>
 
     <!-- jsonld-java depends on depends on Jackson core
-         artifacts so the versions must align. Consult jsonld-java's 
-         POM for the correct dependency versions 
+         and Jackson databind. databind has had a few CVEs 
+         so we exclude from jsonldjava and select the version
```

Comment:
Is that exclusion pre-existing or was that missed from this PR?

Reply:
The dependency is picked up in jena-arq which explicitly pulls in jackson.core. This replaces the jsonld-java ones when they are compatibility (by maven defn).

Can add exclusions if you want. As it stands, if mistakenly jsonld-java is ahead of Jena's, resolution will be the later ones.

See dependency:tree for jena-arq.

Reply:
I would prefer to add the exclusions for clarity... otherwise, if I understand the situation rightly, changing the order of dependencies in a downstream project could create a different dependency tree. That would surprise me!

Reply:
Exclusions for the two jackson artifacts added.